src/azure-cli/azure/cli/command_modules/vm/_vm_utils.py

```diff
@@ -14,6 +14,8 @@
 from knack.log import get_logger
 from knack.util import CLIError
```

Comment:
Can we remove this line since it is not used any more?

Reply:
good catch!